contracts/Zauction.sol

```diff
@@ -31,6 +32,16 @@ contract ZAuction is Initializable, OwnableUpgradeable {
     uint256 expireBlock
   );
 
+  event BuyNow(
+    uint256 auctionId,
+    address indexed buyer,
+    address indexed seller,
+    uint256 amount,
+    address nftAddress,
+    uint256 tokenId,
```

Comment:
Make `tokenid` indexed cause that's a field that someone may want to quickly sort / filter on